raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -104,7 +104,7 @@ contract TokenNetwork is Utils, Controllable {
     struct WithdrawInput {
         address participant;
         uint256 total_withdraw;
-        uint256 expiration_block;
+        uint256 expiration_time;
```

Comment:
This one is actually a different concept, and the name of the variable could probably be changed to `expiration_timestamp` if it would make it more clear.

When we have something like `settle_timestamp`, this refers to the name of seconds from an event until something can be settled, so it's relative time. In this case `expiration_time` means an absolute timestamp, not dependent on any other value.

Reply:
To be fixed in https://github.com/raiden-network/light-client/issues/3015